section1/reflection.md

```diff
@@ -2,18 +2,37 @@
 
 1. Regarding the blog posts in Part A, how do you feel about asking questions? Do you tend to ask them too soon, or wait too long, or somewhere in between?
 
+I have a history of waiting too long to ask questions, but am getting into the habit of asking sooner.
+
 2. What are the data types you learned about in this section? In your own words, define each.
 
+Numbers: Numbers represent number that you would use for your data, using decimal format.
+Strings: A group of words or variables together.
+Boolean: Data that is either true or false.
+
 3. How would you log the string `"Hello World!"` to the console?
 
+console.log("Hello world!");
+
 4. What is/are the character(s) you would use to indicate comments in a JavaScript file? What is the purpose of a code comment?
 
+// You would use code comments to explain any of the code that needs detail.
+
 5. In your own words, what is a variable? How would you explain it to a 5 year old?
 
+A variable is a piece of data to be included in a code. It can indicate anything from a number to an object.
+
 6. Think of a site or app you use frequently. What are three variables that are probably used? Which data type would each of those variables probably hold?
 
+eBay would definitely use number variables. I could see it using Boolean when you put a bid in on an item, to determine if it is higher than the current bid. It most likely uses string variables to welcome you back when you log in.
+
 7. In your own words, explain what concatenation is.
 
+Linking together variables for a code.
+
 8. Think of a site or app you use frequently. Where do you think the developers used concatenation?
 
+I think eBay could use it when it's compiling search results and says "Your search for XXX returned XXX results."
+
 9. What questions do you still have about the work we've done so far? (not having a question is not an option)
+I don't understand what the advantage of concatenation vs interpolation is. It seems to me that they're pretty much the same, just a preference.
```

Comment:
You are spot on! Most devs prefer interpolation these days, they feel it's a bit quicker to type.